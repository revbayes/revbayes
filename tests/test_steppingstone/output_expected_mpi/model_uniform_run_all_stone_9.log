Iteration	Posterior	Likelihood	Prior	alpha	bl[1]	bl[2]	bl[3]	bl[4]	bl[5]	bl[6]	bl[7]	bl[8]	bl[9]	bl[10]	bl[11]	bl[12]	bl[13]	bl[14]	bl[15]	bl[16]	bl[17]	bl[18]	bl[19]	bl[20]	bl[21]	bl[22]	bl[23]	bl[24]	bl[25]	bl[26]	bl[27]	bl[28]	bl[29]	bl[30]	bl[31]	bl[32]	bl[33]	bl[34]	bl[35]	bl[36]	bl[37]	bl[38]	bl[39]	bl[40]	bl[41]	bl[42]	bl[43]	er[1]	er[2]	er[3]	er[4]	er[5]	er[6]	gamma_rates[1]	gamma_rates[2]	gamma_rates[3]	gamma_rates[4]	pi[1]	pi[2]	pi[3]	pi[4]	pinvar	TL
0	-27740.13	-27724.96	-15.16815	1.547743	0.0546478	0.116111	0.0530407	0.115319	0.0781837	0.145179	0.434275	0.231329	0.138412	0.238264	0.310848	0.341106	0.13459	0.0402146	0.0629405	0.197218	0.309392	0.209748	0.150591	0.124875	0.139853	0.170659	0.310758	0.128369	0.0275567	0.0193096	0.122087	0.0943807	0.0168516	0.236987	0.232336	0.217618	0.170083	0.00130904	0.0565618	0.0737826	0.219181	0.169461	0.0164125	0.0411244	0.0382432	0.132662	0.0785258	0.017885	0.275666	0.0652629	0.0273441	0.570426	0.0434159	0.232604	0.596315	1.05311	2.11797	0.140362	0.316513	0.162266	0.380858	0.6272812	6.200398
10	-30002.76	-29999.21	-3.551509	0.7122737	0.147241	0.0866855	0.0345297	0.0753723	0.0567132	0.11075	0.124066	0.114958	0.295273	0.0444517	0.220181	0.140175	0.047679	0.161768	0.134853	0.173349	0.107361	0.092467	0.194661	0.223262	0.100511	0.359073	0.232202	0.126741	0.0855815	0.0566723	0.0274851	0.0463612	0.0134968	0.0933068	0.0860159	0.0916001	0.0774423	0.00208974	0.0947056	0.319018	0.0852798	0.193752	0.00516437	0.0542242	0.0415756	0.114759	0.0785473	0.0279204	0.2007	0.055391	0.00921707	0.659372	0.0473997	0.0768116	0.369264	0.929333	2.62459	0.0511447	0.50422	0.0730503	0.371585	0.009453818	4.971403
