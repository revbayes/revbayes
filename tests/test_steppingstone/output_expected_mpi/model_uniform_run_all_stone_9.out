state	power	likelihood
4	1e-05	-30084.7
5	1e-05	-28407.5
6	1e-05	-28767.2
7	1e-05	-29279.2
8	1e-05	-27847.9
9	1e-05	-28063.7
10	1e-05	-29999.2
